tasks/Makefile.apk

```diff
@@ -25,3 +25,6 @@ apk/clean:
 
 apk: apk/prepare apk/checksum apk/build apk/clean
 	@exit 0
+
+package/prepare: 
```

Comment:
this will not let you overwrite the target. Need `::` at the end to allow it to be overriddent
```
package/prepare::
```